Sources/API/Generated/.openapi-generator/FILES

```diff
@@ -0,0 +1,198 @@
+.gitignore
+.openapi-generator-ignore
+.swiftformat
+Cartfile
+OpenAPIClient.podspec
+OpenAPIClient/Classes/OpenAPIs/APIHelper.swift
```

Comment:
the plural `OpenAPIs` looks odd... 🤷 
I could also see it making sense to put the generated files into a `Generated` folder, so it's easy to see what's handmade and what not?

Reply:
It actually will write all the generated code into `Generated` as you can see in the path of this file but you are right and it seems it does not pick that up for the creation of this `FILES` file

Reply:
ah yes you're right! I don't care too much about this file tbh. What do you guys think?

Reply:
honestly, I didn't even notice this `.openapi-generator` folder and its content 😅 

I don't really think they are needed, for me it looks like a kind of meta info, which is generated every time and everything would work without it... so maybe we can just remove it?